device/iot-device-client/src/test/java/tests/unit/com/microsoft/azure/sdk/iot/device/transport/mqtt/MqttTest.java

```diff
@@ -305,9 +305,8 @@ public void manyExtendsOfAbstractClassDoesNotChangeConfig() throws TransportExce
     @Test(expected = IllegalArgumentException.class)
     public void constructorThrowsExceptionIfConnectionIsNotInitialised() throws TransportException
     {
-        Mqtt mockMqtt = null;
         //act
-        mockMqtt = new MqttMessaging(null, CLIENT_ID, mockedIotHubListener, null, "", "", false, new HashMap<Integer, Message>());
+        Mqtt mockMqtt = new MqttMessaging(null, CLIENT_ID, mockedIotHubListener, null, "", "", false, new HashMap<Integer, Message>());
```

Comment:
can we skip the assignment all together, since the variable is not used?

Reply:
Good idea